code/game/objects/items/cards_ids.dm

```diff
@@ -451,6 +452,7 @@ update_label()
 			update_label()
 			forged = FALSE
 			to_chat(user, "<span class='notice'>You successfully reset the ID card.</span>")
+			log_game("[key_name(user)] has reset [src] to default name \"[registered_name]\" and occupation \"[assignment]\".")

```

Comment:
[src] is now "identification card" and the other two are null.

Reply:
I'll uhhhh actually read the code now and fix that